docs/08-tips-and-tricks.md

```diff
@@ -0,0 +1,121 @@
+# Tips and tricks
+This section explains some tips and tricks to familiarize with OpenShift 4.
+
+## Ignition files
+
+You can use `jq` to read the files easily:
+
+```
+$ jq < master.ign
+{
+  "ignition": {
+    "config": {
+      "append": [
+        ...
+  "storage": {},
+  "systemd": {}
+}
+```
+
+## NTP configuration
```

Comment:
Is chrony not enabled / running / configured by default?

Reply:
It is, but using the default ntp server is not allowed in most environments, instead, a custom one is usually deployed internally and all servers should use it.

Reply:
So can you add that in the description? It's not implicitly clear why you would do this.

Reply:
I've added some clarification.